<ui version="4.0" >
 <class>mainWindow</class>
 <widget class="QTabWidget" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>449</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle" >
   <string>WIDS/WIPS  - Notifier GUI</string>
  </property>
  <property name="tabPosition" >
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape" >
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex" >
   <number>0</number>
  </property>
  <property name="iconSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="elideMode" >
   <enum>Qt::ElideNone</enum>
  </property>
  <widget class="QWidget" name="tabDetection" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <attribute name="title" >
    <string>Detection information</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="boxAlarms" >
      <property name="title" >
       <string>Alarms</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QTextEdit" name="showAlarms" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" >
      <item>
       <widget class="QGroupBox" name="boxStatistics" >
        <property name="title" >
         <string>Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="labelStatActAlarms" >
           <property name="text" >
            <string>Currently raised alarms</string>
           </property>
           <property name="textFormat" >
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="labelShowRaiseAlarms" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelStatTriggAlarms" >
           <property name="text" >
            <string>Triggered alarms</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="labelShowTriggAlarms" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="boxStatus" >
        <property name="title" >
         <string>Status</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelShowStatus" >
             <property name="font" >
              <font>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>No active alarms</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" >
      <item>
       <widget class="QPushButton" name="buttonClear" >
        <property name="text" >
         <string>Clear statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonQuit" >
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabPrevention" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>421</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Prevention information</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <widget class="QGroupBox" name="boxPreventionMsg" >
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title" >
       <string>Prevention messages</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6" >
       <item>
        <widget class="QTextEdit" name="showPreventionMsg" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="boxPreventionCmd" >
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title" >
       <string>Prevention commands</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8" >
       <item>
        <widget class="QTextEdit" name="showPreventionCmd" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="boxPrevStatus" >
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title" >
       <string>Status</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelShowPrevStatus" >
           <property name="font" >
            <font>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>No active prevention actions</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" >
      <item>
       <widget class="QPushButton" name="buttonClearPrev" >
        <property name="font" >
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text" >
         <string>Clear messages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonQuit2" >
        <property name="font" >
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
